Playbooks/Put-MDEAlert-Hunting-GitHub/readme.md

```diff
@@ -0,0 +1,81 @@
+# Tutorial: Push Defender for Endpoint Alerts including Alert Names, MITRE Tactics, Techniques and Sub-Techniques as Hunting ARM Templates into a Sentinel Github Repository 
+
+Intent: As a threat hunter or SecOps analyst, I want to be able to easily hunt for threats pertaining to certain Mitre tactics or techniques in Sentinel. 
+
+Many of these detections are already covered under Defender for Endpoint, and it would be helpful to easily hunt for certain type of Defender for Endpoint alerts by their associated MITRE tactics, techniques and sub-techniques within the Microsoft Sentinel Hunting interface.
+
+   ![Sentinel](./Media/Sentinel-Hunting.png)
+   ![MDE](./Media/MDE-alert.png)
+
+You can see a sample of a generated JSON ARM Hunting Template for Sentinel here - [MDE Alert JSON ARM Hunting Template](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Playbooks/Put-MDEAlert-Hunting-GitHub/Media/MDE_A_process_was_injected_with_potentially_malicious_code_T1055_T1055_001_T1055_002_T1055_003_T1055_004_T1055_012_T1059_001.json)
+
+## Prerequisites
+
+You will need an existing Microsoft Sentinel workspace, connected to a GitHub repository for [Deploying Custom Content from your repository](https://learn.microsoft.com/en-us/azure/sentinel/ci-cd?tabs=github).
+
+You will need [Defender for Endpoint alerts Syncronized to Sentinel](https://learn.microsoft.com/en-us/azure/sentinel/microsoft-365-defender-sentinel-integration) which populates the required SecurityAlerts table in Sentinel. 
+
+You will need to generate a [Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) for your Github Repository associated with Sentinel with the Repo, and Workflow permissions.
+
+You will need your Personal Access Token from GitHub stored in Keyvault and have the Secret Identifier (URL) of the secret handy.
+
+You can retrieve your Secret Identifier (URL) by opening the secret and copying the displayed Secret Identifier value as shown in the example below.
+![Secret](./Media/secret-identifier.png)
+
+This Logic App retrieves the GitHub Personal Access Token from Key Vault using the System Managed Identity and as such you should review and complete the steps in the article [Secure authentication for Conditional Access automation](https://github.com/Azure-Samples/azure-ad-conditional-access-apis/blob/main/00-prereq/readme.md) to create a Key Vault and connect it to a Managed Identity.
+
```

Comment:
@SCStelz , can't we simply assign the access policies to the keyvault which allows the playbook/logic app to fetch the secret instead of going through the longer procedure mentioned in URL above , 

![image](https://user-images.githubusercontent.com/97503740/212249932-eba3d06f-1798-4adf-9d25-f1a2612e5b51.png)

For me assigning the access policies works 

If you agree with this, please update the readme file with screenshot and steps, or let me know your thoughts.